### DISCUSSION TITLE
Duda administrativa sobre el parcial

### DISCUSSION THREAD

**Discussion by ParedesMicaela** [dds-jv/foro#442] [Consultas generales]
Hola buenas, no sé si preguntaron ya esto en la clase de consulta pero se pueden llevar apuntes al parcial? Gracias!

**Comment by flbulgarelli** [comment]
¡Hola!

Sí, no hay problema en tener impresos los apuntes teóricos  que se encuentran [acá](https://dds-jv.github.io/apuntes/)

Saludos
